src/combat.cpp

```diff
@@ -27,36 +27,20 @@
  * \date ????????
  */
 
-#include <cstdio>
-#include <cstring>
-#include <memory>
-
-#include <iostream>
-
 #include "combat.h"
-#include "constants.h"
```

Comment:
Halfway... keep going!
Removing these unnecessary includes is very useful, cuts down the incremental recompilation time quite a bit

Reply:
That's one of my motivations for doing that. Like if anyone changes `kq.h`. Recompiles nearly EVERYTHING.